nvflare/lighter/impl/cert.py

```diff
@@ -79,7 +79,7 @@ def _build_root(self, subject, subject_org):
             self.persistent_state["root_pri_key"] = serialize_pri_key(self.pri_key).decode("ascii")
 
     def _build_write_cert_pair(self, participant, base_name, ctx):
-        subject = participant.subject
+        subject = self.get_subject(participant)
```

Comment:
instead of wrapping "participant.subject" into a method,

suggest just adding type hint `participant: Participant`